eurostag-step-up-transformer/pom.xml

```diff
@@ -68,6 +68,11 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>com.google.jimfs</groupId>
+            <artifactId>jimfs</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Dependencies should be sorted
- by scope (compile, runtime, test)
- by groupId
- by artifactId